src/test/scala/com/google/cloud/spark/bigquery/it/SparkBigQueryEndToEndITSuite.scala

```diff
@@ -96,14 +95,49 @@ class SparkBigQueryEndToEndITSuite extends FunSuite
     spark.read.format("bigquery").option("table", SHAKESPEARE_TABLE).load()
   }
 
+  val filterData = Table(
+    ("condition", "elements"),
```

Comment:
nit: I think
Table(
    "condition" -> "elements",
  ...
is pretty syntactic sugar, but the official docs don't use it so either works for me.
http://www.scalatest.org/user_guide/table_driven_property_checks

Reply:
I'd like to keep this in sync with the official docs -- makes it easier for people to cross-reference the two.